src/Asset.js

```diff
@@ -9,7 +9,6 @@ if (typeof TextDecoder === 'undefined' || typeof TextEncoder === 'undefined') {
     _TextDecoder = encoding.TextDecoder;
     _TextEncoder = encoding.TextEncoder;
 } else {
-    /* global TextDecoder TextEncoder */
```

Comment:
re: this change, see https://github.com/eslint/eslint/issues/11979 and discussion at https://scratchteam.slack.com/archives/CS1144QCE/p1604958351210800